src/Sylius/Bundle/AdminBundle/Tests/Command/ChangeAdminUserPasswordCommandTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Tests\Command;
+
+use PHPUnit\Framework\TestCase;
+use Sylius\Bundle\AdminBundle\Command\ChangeAdminUserPasswordCommand;
+use Sylius\Component\Core\Model\AdminUser;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Sylius\Component\User\Security\PasswordUpdaterInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Tester\CommandTester;
+
+final class ChangeAdminUserPasswordCommandTest extends TestCase
+{
+    private const EMAIL = 'sylius@example.com';
+    private const PASSWORD = 'Password';
+
+    private CommandTester $command;
+    private UserRepositoryInterface $userRepository;
+    private PasswordUpdaterInterface $passwordUpdater;
+
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->userRepository = $this->createMock(UserRepositoryInterface::class);
+
+        $this->passwordUpdater = $this->createMock(PasswordUpdaterInterface::class);
+
+        $this->command = new CommandTester(
+            new ChangeAdminUserPasswordCommand($this->userRepository, $this->passwordUpdater),
+        );
+    }
+
+    /** @test */
+    public function it_does_not_reset_password_when_admin_user_is_not_found(): void
+    {
+        $this->userRepository->expects($this->once())
+            ->method('findOneByEmail')
+            ->willReturn(null);
```

Comment:
```suggestion
        $this
            ->userRepository
            ->expects($this->once())
            ->method('findOneByEmail')
            ->willReturn(null)
        ;
```
?

Reply:
Similar changes could be introduced in the second test

Reply:
Does it look better now?

Reply:
From what I see there're still `;` not in a new line, but ECS will fix that. You can ignore that :p.